components/UI/navbar.tsx

```diff
@@ -37,19 +37,8 @@ const Navbar: FunctionComponent = () => {
     process.env.NEXT_PUBLIC_IS_TESTNET === "true" ? "testnet" : "mainnet";
   const [txLoading, setTxLoading] = useState<number>(0);
   const { hashes } = useTransactionManager();
```

Comment:
I think you can get the hashes directly in the modal instead of getting them here and you won't have to add it as a props